deepchem/feat/molecule_featurizers/circular_fingerprint.py

```diff
@@ -24,6 +25,27 @@ class CircularFingerprint(MolecularFeaturizer):
   Note
   ----
   This class requires RDKit to be installed.
+
+  Examples
+  --------
+  >>> import deepchem as dc
+  >>> from rdkit import Chem
+  >>> smiles = 'C1=CC=CC=C1'
+  >>> mol = Chem.MolFromSmiles(smiles)
+
+  >>> # Example 1: (size = 2048, radius = 4)
+  >>> featurizer = dc.feat.CircularFingerprint(size=2048, radius=4)
+  >>> fingerprint = featurizer([mol])
+  >>> fingerprint.shape
+  (1, 2048)
+
+  >>> # Example 2: (size = 2048, radius = 4, sparse = True, smiles = True)
+  >>> featurizer = dc.feat.CircularFingerprint(size=2048, radius=8,
+  ...                                          sparse=True, smiles=True)
+  >>> featurizer([mol])
+  array([{98513984: {'smiles': 'ccc', 'count': 6}, 2763854213: {'smiles':
```

Comment:
Same comment as above. I think this one might be OK though since CircularFingerprint uses integer math rather than floats but we should check the doctest passes